### DISCUSSION TITLE
Theme for fish shell

### DISCUSSION THREAD

**Discussion by FelipeRosa** [daltonmenezes/aura-theme#124] [Port request]
Really enjoyed the theme! Would love to help to implement a port for fish shell.

**Comment by daltonmenezes** [comment]
Hi @FelipeRosa , it would be great if you do this. You can start by reading the [docs](https://github.com/daltonmenezes/aura-theme#documentation), also you can take a look at [hyper](https://github.com/daltonmenezes/aura-theme/tree/main/src/ports/hyper) and [iTerm](https://github.com/daltonmenezes/aura-theme/tree/main/src/ports/iterm) port implementations to help you, but feel free to ping me if you need.

To upload the fish shell icon, please send a PR to [this repository](https://github.com/daltonmenezes/assets/tree/master/images/icons). 

The preview image I have to work on it, but you can upload the screenshot of fish shell with a good zoom in fullscreen and attach it in your PR here, so I can work on it to apply the right preview model.

It should be something like this, but ignore the effects:
![img](https://github.com/daltonmenezes/assets/raw/master/images/aura-theme/aura-iterm-preview.png?raw=true)

Also, you can add yourself in the contributors section in your port [like this](https://github.com/daltonmenezes/aura-theme/tree/main/packages/rxvt-unicode#contributors)

**Comment by daltonmenezes** [comment]
@FelipeRosa just a few more things.

1. Remember to work on a branch other than the main one, example: `feat/fish-shell`
2. Once the port is done, run `yarn build only port-name` to compile and put it together in your PR, example:
    ```js
    yarn build only fish-shell
    ```
That's it 💜

**Comment by daltonmenezes** [comment]
@FelipeRosa Now the [Publishing ports](https://github.com/daltonmenezes/aura-theme/blob/main/docs/PUBLISHING_PORTS.md) doc is available 🚀

**Comment by FelipeRosa** [comment]
Cool! Thanks a lot for the helpful tips @daltonmenezes  :smile: 

I'll start working on something today.

**Comment by daltonmenezes** [comment]
Hi @FelipeRosa , are you still working on this port for fish shell?

**Reply by FelipeRosa** [reply]
Hey @daltonmenezes, I'm not. I got carried away with some work related stuff.
